PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,119 @@
+- [ ] I am not reading every word and I need to start this checklist over.
+    - Please uncheck everything you have checked and start over.
+- [ ] Location of connectors and wires going out of the board will prevent messy wiring.
+    - Are all the wires that are going in the same direction placed on the same edge of the board?
+
+### Components
+- [ ] Do footprints match the schematic?
+    - Make sure pins on the footprint correspond to pins in the schematic 
+    - (e.g. counterclockwise pins on footprint and alternating pins on schematic is *bad*)
+- [ ] Custom footprints have been double checked with the datasheet.
+- [ ] Footprints are consistent across the PCB: *i.e. all 0805 for Resistors/Capacitors*
+- [ ] Pin 1 of the footprint is labeled in some way. 
+- [ ] Are LED's in easy to see places? 
+- [ ] Are test points in easy to reach places?  
+- [ ] Are critical paths of switching converters as small as possible?  
+
+### Copper Layer 
+- [ ] The trace widths and trace clearances are greater than JLCPCB's minimum requirements. 
+    - [ ] Are signal traces 6mils unless provided with reasoning?   
+        - NOTE: One net can have multiple trace widths  
+- [ ] The trace lengths are as short as possible. 
+    - Can there be a more optimal route if you go to another layer? 
+- [ ] Each trace's width is capable of handling the expected current flow. 
+    - Use PCB width calculator to calculate trace width. 
+- [ ] *No sharp corners. No trace angles equal to or less than 90 degrees. 
+    - Orthogonal traces should have vias if necessary. 
+- [ ] Are edges of board surrounded by clean ground on both layers with stitching vias?  
+- [ ] Traces are in parallel with each other when possible. 
+    - E.g. traces connected between an IC and MCU can be placed in parallel with each other. 
+- [ ] There are no random trace appendages. 
+- [ ] Vias placed in copper pads are fully encompassed in the copper pads. 
+- [ ] Through-hole components do not have extraneous vias. 
+- [ ] Vias should be appropriately sized according to the connected net(s) and expected current capacity.
+
+*Not really a problem for modern manufacturing techniques but good practice and important for high speed signal integrity.
+
+### Silkscreen Layer
+- [ ] Visible text sizes are greater than JLCPCB's minimum requirements.
+- [ ] All visible text is on the silkscreen layer.
+- [ ] All reference labels of each component are not overlapping a copper pad or another component.
+- [ ] All connector pins are labeled with a meaningful and helpful name.
+- [ ] All LEDs are labeled with a meaningful and helpful name.
+- [ ] Silkscreens are mostly facing one direction (or 2). 
+
+### Edge Cut Layer
+- [ ] The dimensions of the board and the mounting holes are nice values in metric i.e. no long decimals.
+- [ ] The physical outline of the board is on the edge cut layer.
+- [ ] Edge cuts are straight and parallel with opposite edge of the board.
+- [ ] Mounting holes are aligned.
```

Comment:
Is there a LHR standard on mounting holes? Are we all doing M2, M2.5, M3 screws or something?
Also would like some layout documentation, perhaps in the comments layer dimensioning holes that may be located not on the edges of the board but in the middle, for the mechanical people.

Reply:
All M3, and good point.